style.css

```diff
@@ -65,3 +81,168 @@
   padding: 0 15px 0 0;
   list-style: none;
 }
+
+.works {
+  display: grid;
+  grid-gap: 80px;
+  padding: 130px 24px;
+}
+
+.works-project {
+  grid-template-columns: 1fr;
+  border-radius: 8px;
+  display: flex;
+  flex-direction: column;
+  padding: 16px;
```

Comment:
- [ ] **Required** Let's tweak the design a bit to have equal `margin` or `padding` on both left and right. the applied adding seems not to work so well on the design. Use the developer tool mobile screen as your guide rather than shrinking your browser, try to make it consistent through small, medium, and large mobile screens. I hope it makes sense.

**Your design** 

![image](https://user-images.githubusercontent.com/27709832/143621548-afc357e2-06c5-446f-83bd-99833cdb2812.png)

**Figma design** 

![image](https://user-images.githubusercontent.com/27709832/143621579-0eaa959c-76ed-40e3-9c5e-8d5abb8f997b.png)